src/modules/poweraccent/PowerAccent.Core/PowerAccent.cs

```diff
@@ -134,26 +134,51 @@ private void ProcessNextChar(TriggerKey triggerKey)
 
         if (triggerKey == TriggerKey.Space)
         {
-            if (_selectedIndex < _characters.Length - 1)
+            if (WindowsFunctions.IsShiftState())
             {
-                ++_selectedIndex;
+                if (_selectedIndex < 0)
+                {
+                    _selectedIndex = _characters.Length - 1;
+                }
+                else
+                {
+                    --_selectedIndex;
+                }
             }
             else
```

Comment:
We'd prefer not to include this behavior, since pressing shift on the context of inserting a character means capitalizing it. Please remove it.

Reply:
Done!